### PR TITLE
Docker image for cloud code

### PR DESCRIPTION
**Pull request by cheungpat**
connects #94

### CODE REVIEW

**Comment by oursky-travis** [comment]
@cheungpat, thanks for your PR! By analyzing the annotation information on this pull request, we identified @rickmak to be a potential reviewer

**Comment by cheungpat** [comment]
@rickmak your move

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      dockerfile: Dockerfile-development
+    command: skygear-node
+    volumes:
+    - .:/usr/src/app
+    environment:
+      DATABASE_URL: postgres://postgres@db/postgres?sslmode=disable
+      API_KEY: changeme
+      MASTER_KEY: secret
+      APP_NAME: _
+      PUBSUB_URL: PUBSUB_URL=ws://app:3000/pubsub
```
Comment (ben181231) [inline docker-compose.yml]: Duplication of `PUBSUB_URL` ?

Comment (cheungpat) [inline docker-compose.yml]: interesting i copied it from elsewhere

**Comment by cheungpat** [comment]
@ben181231 fixed

**Pull request merged**